test/common/testlib.py

```diff
@@ -672,14 +675,14 @@ def open_session_menu(self):
             self.click("#toggle-menu")
 
     def open_superuser_dialog(self):
-        if self.cdp.mobile:
+        if self.current_layout == "mobile":
             self.open_session_menu()
             self.click("#super-user-indicator-mobile button")
         else:
             self.click("#super-user-indicator button")
 
     def check_superuser_indicator(self, expected):
-        if self.cdp.mobile:
+        if self.current_layout == "mobile":
```

Comment:
...then, for example

```suggestion
        if self.current_layout.is_mobile:
```